package.json

```diff
@@ -42,6 +42,7 @@
     "prettier:all": "yarn babel-node ./scripts/prettier.js write",
     "size": "size-limit",
```

Comment:
This command does no longer work.

```
 ERROR  Can not find settings for Size Limit.
        Add it to section "size-limit" in package.json according to Size Limit docs.

  "size-limit": [
    {
      "path": "index.js",
      "limit": "9 KB"
    }
  ]

error Command failed with exit code 1.
```

Reply:
Maybe we can kill it?

Reply:
I think so. Maybe we can abstract some of the dangerfile into a local task so that devs can get a size comparison of their local work.

Reply:
I'm not sure it's needed. People can build the packages with Lerna, then the docs, then run the snapshot generation.